code/API_definitions/connected-network-type-subscriptions.yaml

```diff
@@ -554,14 +539,14 @@ components:
       required:
         - device
 
-    SubscriptionEventType:
+    EventTypeNotification:

```

Comment:
I think that's the problem. Now EventTypeNotification doesn't have the subscription-ends event and SubscriptionEventType has that, but it shouldn't